CMSgov/onemac
# implement vertical nav for detail view

Story: https://qmacbis.atlassian.net/browse/OY2-14656
Endpoint: TBD

### Details

Implement a vertical navigation element in our package/component detail view.

### Changes

- Vertical tabbed navigation on the left side of the detail view

### Implementation Notes

- Some typings cleanup in both DetailView and its dependencies, and PortalTable

### Test Plan

1. Log in as `statesubmitteractive` and go to Packages
2. Click on a package or child component
3. Verify that the nav is present and looks sort of like the Figma. Note that there should only be one item, so functionality will not really be testable until the next story comes through.
